pkg/cmd/cli/cmd/volume.go

```diff
@@ -28,50 +31,52 @@ const (
 Update volumes on a pod template
 
 This command can add, update or remove volumes from containers for any object
-that has a pod template (replication controllers or deployment configurations).
+that has a pod template (deployment configs, replication controllers, or pods).
```

Comment:
Once pods are created, we can't update the volumes right? I thought we can only modify volumes through dc/rc.